### PR TITLE
Fix/bug 1

### PR DESCRIPTION
**Pull request by aatbip**
Added 2nd fix for production

### CODE REVIEW

File: src/app/components/EditorInterface.tsx [hunk 1]
```diff
@@ -228,9 +229,24 @@ const EditorInterface = ({ settings, token }: IEditorInterface) => {
...
+            url: '',
+            filename: '',
+            contentType: '',
+            size: 0,
+            createdById: '',
+          },
+          createdById: '',
+        }
+        appState?.setOriginalTemplate(settings?.content || '')
+        appState?.setSettings(settings ? settings : _settings)
```
Comment (sazanrjb) [inline src/app/components/EditorInterface.tsx:249]:
```suggestion
        appState?.setSettings(settings ||  _settings)
```

File: src/app/components/EditorInterface.tsx [hunk 2]
```diff
@@ -243,7 +259,7 @@ const EditorInterface = ({ settings, token }: IEditorInterface) => {
       (appState?.appState.settings as ISettings).backgroundColor,
     )
     appState?.setBannerImgUrl(
-      (appState?.appState.settings as ISettings).bannerImage.url,
+      (appState?.appState.settings as ISettings).bannerImage?.url || '',
```
Comment (sazanrjb) [inline src/app/components/EditorInterface.tsx:262]: Lets avoid typecasting when possible, we can enforce the type with Zod. Can be done later

File: src/app/components/EditorInterface.tsx (hunk 1 above)
Comment (aatbip) [reply src/app/components/EditorInterface.tsx]: resolved in latest commit

File: src/app/components/EditorInterface.tsx (hunk 2 above)
Comment (aatbip) [reply src/app/components/EditorInterface.tsx:262]: left this for later

**Review by sazanrjb — APPROVED** [review]

**Pull request merged**